net/udp-broadcast-relay-redux-openwrt/files/udp_broadcast_relay_redux.config

```diff
@@ -0,0 +1,6 @@
+#config udp_broadcast_relay_redux
```

Comment:
should this filename use dashes instead of underscores?

Reply:
Both names are okay :-)
But... Why do you comment out all of the config?